src/features/selectedPost.ts

```diff
@@ -0,0 +1,24 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+import { Post } from '../types/Post';
+
+type State = {
+  postData: Post | null,
+};
+
+const initialState: State = {
+  postData: null,
+};
+
+const selectedPostSlice = createSlice({
+  name: 'selectedPostSlice',
+  initialState,
+  reducers: {
+    setPost: (state: State, action: PayloadAction<Post | null>) => {
+      // eslint-disable-next-line
+      state.postData = action.payload;
+    },
+  },
+});
+
+export default selectedPostSlice.reducer;
+export const { setPost } = selectedPostSlice.actions;
```

Comment:
consider adding a `selectedPost` field to the `posts` feature

Reply:
ok, thanks, it will be more logical